dev-python/subprocess32/subprocess32-3.2.7.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="A backport of the subprocess module from Python 3.2/3.3 for use on 2.x"
+HOMEPAGE="https://github.com/google/python-subprocess32"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="PSF-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc64 ~x86 ~amd64-linux ~ppc-macos ~x64-macos ~x86-macos"
```

Comment:
I'd originally had fewer here but my matplotlib ebuild was broken since it had all of the keywords. I've dropped the keywords on both the new matplotlib ebulid and this ebuild to what I have access to `~x86` and `~amd64`